packages/esm-patient-medications-app/src/add-drug-order/add-drug-order.workspace.tsx

```diff
@@ -49,6 +49,17 @@ export default function AddDrugOrderWorkspace({
 
   const saveDrugOrder = useCallback(
     (finalizedOrder: DrugOrderBasketItem) => {
+      // a prper error message instead of crushing
+      if (!session.currentProvider) {
+        // Show a proper error message instead of crashing
+        alert(
```

Comment:
Never use `alert()`. Please use the framework's notification methods to send an error message.